### DISCUSSION TITLE
2023-10-06 - Sydney Brainstorming #2 - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by piers-sinclair** [SSWConsulting/SSW.Rules.Content#6898] [Chewing the Fat]
Hey SSW'ers

Let's talk about brainstorming!

**Comment by jernejk** [comment]
I’m very excited for this brainstorming (unfortunately not joining as I’m not in Sydney), lots of ideas I pitched for years now are pitched by others. 😊

**Comment by matt-goldman** [comment]
Some cool ideas this year, looking forward to seeing the presentations :)

**Comment by 2fernandez** [comment]
![0*0Nr50tfYU5rjm6g_](https://github.com/SSWConsulting/SSW.Rules.Content/assets/27670973/51653098-1e93-4e03-9c48-029d80acb6d5)

**Reply by wicksipedia** [reply]
mrotsniarb?

**Comment by Ben0189** [comment]
I'm excited for the upcoming brainstorming day

**Comment by drwharris** [comment]
Some tough calls to make on what not to do.

**Comment by UlyssesMaclaren** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/45545178/9a725526-9f87-4379-93a8-c3948bd9d21c)
Figure: my artistic impression of what happens during a brainstorming session. thanks Dall-e 3 :)

**Reply by wicksipedia** [reply]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/600044/018feb8a-617a-4df7-9082-1ad77465b662)
Figure: After Uly has all his AI powers

**Comment by apforsyth** [comment]
Sometimes inspiration hits while you are noodling in the shower - also when you are on a long run or a bikeride.  I sometimes write it down and the next day and surprised by ridiculous thoughts that pop into my head - then go, wait!  we can do it, let's make it happen!

The trick it to notice your brain when you're idling... anyone else get that some thing?

**Comment by Levijj22** [comment]
So many great ideas! Looking forward to it.

**Comment by MarcusIrmscherSSW** [comment]
Looking forward to watching the magic happen.
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/129343825/25188375-58c6-4c30-98ea-7655ad8e3cab)

**Comment by joshbermanssw** [comment]
Some helpful ideas floated around

**Comment by Calinator444** [comment]
I like the idea of AI being used to help with auditing which rules get posted and minimizing duplicate rules. If it's not a feature already I think it would also be a good idea to send an email to the author of the rule when changes are suggested to make sure they get approved.

**Comment by cvionnet** [comment]
Lot's of great ideas!

**Comment by Marxoz** [comment]
Lots of great ideas. See you there :)

**Comment by ricksu978** [comment]
I wanna pick more than 3 options!

### Suggestion - Create the question using "Ranking"
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/35659961/9c608936-7812-416d-9fd5-4f63c676e387)
Figure: The "Ranking" option allows users to rank their favorite projects.

### And this is how it looks when filling out the form.
https://github.com/SSWConsulting/SSW.Rules.Content/assets/35659961/bf503f07-d415-4ed3-a2c0-fba4aaa01233

**Reply by jontygardner** [reply]
+ 1 to this Rick!

**Comment by BenDunkerleySSW** [comment]
Keen to see people with different skills collaborate! Lots of good ideas that encourage lateral thinking.

**Comment by Dhruv-0987** [comment]
Wow just saw the ideas they were amazing, particularly market and sell our internal products hoping for this one to get a go ahead.

**Comment by sethdaily** [comment]
Better format for voting ideas than last time

**Comment by SamBWagner** [comment]
Hope everyone has fun at the brainstorming!

**Comment by jeoffreyfischer** [comment]
Working more with AI seems like the topics that we should be focusing on these days.
I also agree that we compile and summarize SSW knowledge database, to improve future efficiency.
Finally, having an app to check email formatting would also improve work efficiency.

**Comment by tkapa** [comment]
Make something good >:(

And implement the new megamenu >:(

**Reply by wicksipedia** [reply]
**cough** https://github.com/SSWConsulting/SSW.MegaMenu/tree/menu-redesign **cough**

there are a few things to think/plan out before firing it up

**Comment by Jord-Gui** [comment]
Looking forward to it!

**Comment by vladislav-kir** [comment]
Looking forward to see what will come out from these ideas :)

**Comment by Abf09** [comment]
Some ideas are really interesting, specially Semantic Kernel hackday - Work on Timesheet GPT. Good to see many AI related topics for brainstorming day.

**Comment by AntPolkanov** [comment]
Interesting ideas! It's sad though that _Automating Sick Day Emails & Appointments_ was dropped 😕

**Reply by wicksipedia** [reply]
I think Easy Leave (brainstorming idea from a previous year) has had the green light (to some extent) and will cover this functionality

**Comment by leahy268** [comment]
Lot's of interesting stuff there.

**Comment by amankumarrr** [comment]
Looking forward to seeing these ideas implemented, especially the ones related to marketing SugarLearning and TimePro.

**Comment by JackDevAU** [comment]
First time going to Sydney!! 
Can't wait to meet everyone and expand on these ideas!!!

**Comment by steven0x51** [comment]
Excited and looking forward to it

**Comment by bahjatalaadelssw** [comment]
I'm looking forward to more leads and videos.

**Comment by tomek-i** [comment]
I like 🍖 - looking forward to some picaña 
Curios about Go-karting ... I heard lots of stories ... 

Great idea line-up

**Comment by AshAnil** [comment]
Looking forward to Brainstorming 🥳 I am sure we can learn a lot 📖

**Comment by zacharykeeping** [comment]
Some good ideas there - good luck to everyone at the Brainstorming

**Comment by Harry-Ross** [comment]
Looking forward to the brainstorming day - a lot of focus on AI which has got me excited! 😄

**Comment by NadeekaJK** [comment]
Looking forward to it!!!

**Comment by jimmy1992abc** [comment]
I cannot join the Brainstorming this time due to an exam that takes place the same day😥

Can someone brainstorm the idea of having a race simulator 📺🏎️🧑‍🦼 in the office? 

It is a big step to boost productivity!

**Comment by MattParkerDev** [comment]
🧠⛈️

**Comment by tombui99** [comment]
Some really cool ideas that makes me very excited to see what the outcomes will be!

**Comment by stephanfakossw** [comment]
Looking forward to it. There is some great ideas out there!

**Comment by CoffeeBugHub** [comment]
A lot of cool ideas! Brainstorming is a fun event!

**Comment by PennyWalker** [comment]
There are so many good ideas I had trouble choosing 3, so I picked ones that help me... mu ha ha ha

**Comment by jackreimers** [comment]
Lots of great ideas, looking forward to participating in my first brainstorming!

**Comment by ozair-ashfaque** [comment]
Some are cool ideas, looking forward to it

**Comment by jontygardner** [comment]
Great ideas!

+ 1 to Rick's suggestion. This should be a ranking of preferred choices and not just pick your top 3. 

#Democracy 😂

**Comment by babakamyljanovssw** [comment]
Great ideas! It was hard to choose 3 ideas.
Totally agree with optimizing SSW rules, and removing outdated or overlapped rules now before it becomes late and not maintainable.

**Comment by BrookJeynes** [comment]
Some cool ideas being passed around, interested to see what gets chosen

**Comment by danielmackay** [comment]
Love the Inbox Zero, TimesheetGPT, and Spec Review AI ideas.  Very interested to see what these result in!

**Comment by chrisschultzssw** [comment]
looking forward to it!
